scripts/tests/test_publish_infra_k8s_outputs.py

```
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+
+import pytest
+
+REPO_ROOT = Path(__file__).resolve().parents[2]
+if str(REPO_ROOT) not in sys.path:
+    sys.path.insert(0, str(REPO_ROOT))
+
+from scripts.publish_infra_k8s_outputs import (  # noqa: E402
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)